instructure/canvas-ios
# Work around using customData because of a pspdfkit bug.

refs: MBL-15825
affects: Student, Teacher
release note: none

test plan:
- Create an assignment with a pdf file attached.
- As a student, download the pdf file to iOS Files.
- Open the pdf in iOS Files and add annotations.
- Submit the annotated file to the assignment.
- As a student check your submission in Submission & Rubric.
- As a teacher go to SpeedGrader and check the file.
- Annotations added in Files app should appear.
- As a teacher single tap on this file annotation, context menu shouldn't appear.
